src/reanimated2/helperTypes.ts

```diff
@@ -83,7 +83,7 @@ type AnimatedStyleProps<Props extends object> = {
   [Key in keyof PickStyleProps<Props>]: StyleProp<AnimatedStyle<Props[Key]>>;
 };
 
-type NonStyleAnimatedProps<Props extends object> = {
```

Comment:
I just hated the name.

Reply:
Can you add a comment describing precisely what are this Props used for? Are they all animated props that are not style? Or something different?